experimental/dds/tree2/package.json

```diff
@@ -11,6 +11,38 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/tree2-untrimmed.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/tree2-untrimmed.d.ts",
+				"default": "./dist/index.js"
+			}
```

Comment:
I used the untrimmed types here, and I wanted to use them in the top-level types field too. Unfortunately that breaks compilation of some packages, for reasons I don't yet understand. So I left the top-level one as-is. The exports values don't seem to be respected unless the typescript moduleResolution is node16, so no other packages see the untrimmed types yet.